presto-main/src/main/java/io/prestosql/sql/planner/PlanOptimizers.java

```diff
@@ -91,6 +91,7 @@
 import io.prestosql.sql.planner.iterative.rule.RemoveFullSample;
```

Comment:
please add a test to `TestLogicalPlanner` and `ATQ`

Reply:
Test has been added for it